README.md

```diff
@@ -1,12 +1,74 @@
-# Repo for SRE Content
+# Operate First - Site Reliability Engineering
 
-##  Material
+## Introduction
 
-* [Architecture Decision Records](./ADRs/)
-* [SRE Maturity Model](./sre_maturity.md)
-* [Picking Good SLIs](./picking_good_slis.md)
+This the Operate First repository for Site Reliability Engineering (SRE)
+documentation. Here we intend to collect documentation on the nitty-gritty
+aspects of building and operating an effective SRE organisation.
+
+## Who runs this repository?
+
+Hello! We're the Red Hat SIG-SRE group, a cross-team group of people inside Red
+Hat with an interest in promoting and developing strong SRE culture. As a
+result you can expect some of the material here to be a little slanted toward
+the way we do things here inside Red Hat, but as we're doing our best to stick
+to what are generally accepted to be good SRE practices you should be able to
+make use of it with, at most, minor modification.
+
+##  Table of Contents
+
+### Metrics and Monitoring
+
+* [Picking good Service Level Indicators (SLIs)](./picking_good_slis.md)
+
+    Service Level Indicators are one of the most important sets of metrics
+    in SRE, and defining them correctly is key to running an effective
+    SRE service.
+
+* [Picking good Service Level Objectives (SLOs)](./picking_good_slos.md)
+
+    Where SLIs are used to answer the question "Is everything working as it
+    should?", Service Level Objectives (SLOs) define the expectations for
+    how much and how often a service should perform correctly according
+    to its SLIs. They are probably the single most important set of
+    statistics used for evaluating the performance of a managed service,
+    as they show maintainers and customers alike just how well things are
+    working.
+
+* [Prometheus Alerting Consistency](./prometheus_alerting_consistency.md)
+
+    Alerts are only useful if they're clear and understandable. This guide
+    presents some suggested best practices for designing alerts that are
+    consistent and useful.
+
+### Decision-Making
+
+* Some sample SIG-SRE [Architecture Decision Records](./ADRs/RH/SIG-SRE)
+
+    When major decisions are made on design or operational aspects of
+    system, it can be very worthwhile to keep a clear record of those
+    decisions for future reference. An Architecture Decision Record
+    (ADR) acts as a document of the discussion and reasoning behind the
+    decision it relates to, and in years to come can be extremely valuable
+    for answering the eternal question - "Why do we do things this way?".
+
+    This directory contains some sample Red Hat ADRs from a variety of
+    different discussions.
```

Comment:
I think here we'd want to clarify that the ADRs are specific to SIG-SRE topics instead of a general set of Red Hat ADRs.

Reply:
PTAL